projects/Amlogic/devices/Odroid_C2/bootloader/user.ini

```diff
@@ -0,0 +1,112 @@
+#------------------------------------------------------------------------------------------------------
```

Comment:
I only have 1 suggestion, perhaps this should be called boot.user.ini as user.ini is somewhat vague.

and then add a header to boot.ini saying do not modify, changes may be lost.

Reply:
The header is fine. I'm not sure about the user ini filename yet.

Reply:
pi has config.txt so perhaps config.ini ?